packages/protocol/test/governance/vesting.ts

```diff
@@ -240,9 +244,8 @@ contract('Vesting', (accounts: string[]) => {
     })
 
     it('should have associated funds with a schedule upon creation', async () => {
-      await createNewVestingInstanceTx(vestingDefaultSchedule, web3)
-      const vestingInstanceRegistryAddress = await vestingFactoryInstance.vestings(beneficiary)
-      const vestingInstance = await VestingInstance.at(vestingInstanceRegistryAddress)
+      vestingInstanceAddress = await vestingFactoryInstance.vestings(beneficiary)
```

Comment:
Why do you need to set these variables again?

Reply:
fixed